src/libraries/System.Private.Xml/tests/System.Private.Xml.Tests.csproj

```diff
@@ -621,7 +621,8 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="(('$(TargetOS)' != 'tvOS' and '$(TargetOS)' != 'iOS') or '$(RunDisablediOSTests)' == 'true') and
+                        ('$(TestNativeAot)' != 'true' or '$(RunDisabledNativeAotTests)' == 'true')" >
```

Comment:
It might be better to put `[ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsReflectionEmitSupported))]` on the individual test classes in these files. It's more consistent with how we do things and more futureproof.